lua/ulx/modules/sh/cfc_constraint_checker.lua

```diff
@@ -0,0 +1,129 @@
+CFCUlxCommands.constrainChecker = CFCUlxCommands.constrainChecker or {}
+local cmd = CFCUlxCommands.constrainChecker
+local CATEGORY_NAME = "Utility"
+local IsValid = IsValid
+
+-- Returns the owner of the entity if it is checkable, otherwise returns false
+local function getOwnerIfCheckable( ent )
+    if not IsValid( ent ) then return false end
+    if ent.IsConstraint and ent:IsConstraint() then return false end
+
+    local owner = ent.CPPIGetOwner and ent:CPPIGetOwner() or ent:GetOwner()
+    if not IsValid( owner ) then return false end
+
+    return owner
+end
+
+-- Returns a lookup table from players to the entities they own
+local function getCheckableEnts( plys )
+    local allEnts = ents.GetAll()
+    local perPlyEnts = {}
+
+    for i = 1, #plys do
+        local ply = plys[i]
+        perPlyEnts[ply] = {}
+    end
+
+    for i = 1, #allEnts do
```

Comment:
Use `ipair` and table.insert instead

Reply:
Replaced with `ipair`, but `table.insert` only takes a number for the index the value is inputted in. Unless there is a way to make `perPlyEnts[ply] = {}` in the `table.insert` format, I think the way it is, is the only way.

Reply:
i mean for `ownedEnts[#ownedEnts + 1] = ent` not for `perPlyEnts`